spec/rubocop/cop/rspec/no_stringified_instance_double_constant_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::NoStringifiedInstanceDoubleConstant,
+               :config do
+  context 'when using a class for instance_double' do
+    let(:source) do
+      <<~RUBY
+        instance_double(Foo, failure_message, stubs)
+      RUBY
+    end
+
+    it { expect_no_offenses source }
+  end
```

Comment:
I much prefer to have specs written out, especially in simple cases like this.
```suggestion
  it 'finds no offense when using a class for instance_double' do
    expect_no_offenses(<<~RUBY)
      instance_double(Foo, failure_message, stubs)
    RUBY
  end
```

Reply:
Thanks!